### DISCUSSION TITLE
App Installer

### DISCUSSION THREAD

**Discussion by S1-Othonk** [phoenixbyrd/Termux_XFCE#68] [Q&A]
![Screenshot_20241114-160643_TermuxX11.jpg](https://github.com/user-attachments/assets/899a9cbf-9641-4bb5-801a-ebfccf75c29f)

The app installer shown like that. How to fix it?

**Comment by phoenixbyrd** [comment]
I never even noticed discussions existed ... Anyways, idk what happened but the issue seems to be between how zenity and hardware acceleration work and I'm not sure how to resolve. 

I'll be pushing out a new installer eventually for additional apps. No eta on that though ... As soon as I can get it done in between everything else I have to do.
